src/Components/Forms/src/EditContextDataAnnotationsExtensions.cs

```diff
@@ -94,6 +94,7 @@ private static void ValidateField(EditContext editContext, ValidationMessageStor
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2080", Justification = "This is not a trimmer safe pattern.")]
```

Comment:
This shouldn't be suppressed. But instead have `RequiresUnreferencedCode` attribute.